docs/managing_deis/running-stateless-cluster.rst

```diff
@@ -0,0 +1,214 @@
+                                  s3accessKey=${AWS_ACCESS_KEY} \
+                                  s3secretKey=${AWS_SECRET_KEY} \
+                                  s3region=eu-west-1 \
+                                  s3path=/ \
+                                  s3encrypt=false \
+                                  s3secure=false
+    $ deisctl config store set gateway/accessKey=${AWS_ACCESS_KEY} \
+                               gateway/secretKey=${AWS_SECRET_KEY} \
+                               gateway/host=s3.amazonaws.com \
+                               gateway/port=80
+
+Configure database settings
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Since we won't be running the :ref:`database`, we need to configure these settings
+so the controller knows where to connect.
+
+.. code-block:: console
+
+    $ HOST=something.rds.amazonaws.com
+    $ DB_USER=deis
+    $ DB_PASS=somethingsomething
+    $ DATABASE=deis
+    $ deisctl config database set engine=postgresql_psycopg2 \
+                                  host=${HOST} \
+                                  port=5432 \
+                                  name=${DATABASE } \
+                                  user=${DB_USER} \
+                                  password=${DB_PASS}
+
+Fake deis-store-volume
+^^^^^^^^^^^^^^^^^^^^^^
+
+The ``deis-store-volume`` component mounts the shared Ceph filesystem used by
+both the :ref:`logger` and :ref:`controller`. For the controller to run, it needs
+to think that the log directory exists, so we create a fake ``deis-store-volume``
+component.
+
+This code snippet is intended to be run on a CoreOS host because it uses
+``fleetctl`` directly.
+
+.. code-block:: console
+
+    $ cat << EOF > /tmp/deis-store-volume.service
+    [Unit]
+    Description=dummy deis-store-volume
+    [Service]
+    EnvironmentFile=/etc/environment
+    ExecStart=/usr/bin/mkdir -p /var/lib/deis/store/logs
+    RemainAfterExit=yes
+    Type=oneshot
+    [Install]
+    WantedBy=multi-user.target
+    [X-Fleet]
+    Global=true
+    EOF
+    $ fleetctl load /tmp/deis-store-volume.service
+    $ fleetctl start deis-store-volume.service
+    $ rm /tmp/deis-store-volume.service
+
+Deploy the platform
+^^^^^^^^^^^^^^^^^^^
+
+The typical :ref:`install_deis_platform` documentation can be followed, with
+one caveat: since we won't be deploying many of the typical Deis components, we cannot
+use ``deisctl install platform`` or ``deisctl start platform`` -- instead, we
+use ``deisctl install stateless-platform`` and ``deisctl start stateless-platform``.
+
+These commands tell ``deisctl`` to skip the components that we don't need to use.
+
+Confirm installation
+^^^^^^^^^^^^^^^^^^^^
+
+That's it! Deis is now running without Ceph. Issue a ``deisctl list`` to confirm
+that the services are started, and see :ref:`using_deis` to start using the cluster.
+
+Upgrading Deis
+--------------
+
+When following the :ref:`upgrading-deis` documentation, be sure to use
```

Comment:
> For inplace upgrade it might.

That's the recommended upgrade path, so we should recommend that for stateless upgrades as well. If there's an issue with an in-place upgrade for stateless clusters, we should see if we can fix that.

I'm not sure when someone would prefer to ever deploy a new cluster and cut over as your instructions suggest when they can instead just upgrade in-place, unless I'm missing something?